app/components/views/TrezorPage/TrezorPage.jsx

```diff
@@ -0,0 +1,21 @@
+import { StandalonePage, StandaloneHeader } from "layout";
+import { FormattedMessage as T } from "react-intl";
+import TrezorPageContent from "./TrezorPageContent";
+
+const TrezorPageHeader = () => (
+  <StandaloneHeader
+    iconClassName={"trezor"}
```

Comment:
can this use a new module's class instead of old `.less` class ?

Reply:
It's defined at ``TitleHeader.module.css`` which is imported by ``TitleHeader.jsx``.

Reply:
Oh, I see.
Yea I remember seeing this earlier, if you like you could change the prop to be `iconType` instead of `iconClassName`, which might be more accurate as we aren't passing a class here.

Reply:
I really prefer to implement that at a new PR, so I opened a new issue to solve it https://github.com/decred/decrediton/issues/3295.

Reply:
Sweet, thank you